### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by martialblog**
Extend README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,6 +60,12 @@ go check.HandleTimeout(checkPluginTimeoutInSeconds)
 
 ## Thresholds
 
+Threshold objects represent Nagios-style thresholds that have methods to evaluate if a given input is within the range.
```
Comment (RincewindsHat) [inline README.md:63]: Entirely not Nagios related

File: README.md [hunk 2]
```diff
@@ -60,6 +60,12 @@ go check.HandleTimeout(checkPluginTimeoutInSeconds)
...
+
+They can be created with the ParseThreshold parser.
+
+https://nagios-plugins.org/doc/guidelines.html#THRESHOLDFORMAT
```
Comment (RincewindsHat) [inline README.md:67]: Please link to this or the RFC draft: https://github.com/monitoring-plugins/monitoring-plugin-guidelines/blob/main/definitions/01.range_expressions.md

File: README.md [hunk 3]
```diff
@@ -74,6 +80,10 @@ if warnThreshold.DoesViolate(3.6) {
 
 ## Perfdata
 
+The Perfdata object represents Nagios-style performance data that relates to the actual execution of a host or service check. 
+
+https://nagios-plugins.org/doc/guidelines.html#AEN200
```
Comment (RincewindsHat) [inline README.md:85]: I prefer this https://github.com/monitoring-plugins/monitoring-plugin-guidelines/blob/main/monitoring_plugins_interface/03.Output.md#performance-data

**Review by RincewindsHat — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (martialblog) [reply README.md]: That's just me being old. I changed it.

File: README.md (hunk 2 above)
Comment (martialblog) [reply README.md]: Done

File: README.md (hunk 3 above)
Comment (martialblog) [reply README.md]: Done

**Pull request merged**